tutor/src/theme.js

```diff
@@ -183,10 +183,10 @@ export const breakpoint = {
   },
   only: {
     mobile(...args) {
-      return css`@media(max-width: 599px) { ${css(...args)} }`;
+      return css`@media(max-width: 600px) { ${css(...args)} }`;
     },
     tablet(...args) {
-      return css`@media(min-width: 600px) and max-width: 1199px) { ${css(...args)} }`;
+      return css`@media(min-width: 601px) and max-width: 1199px) { ${css(...args)} }`;
```

Comment:
@jivey it is missing a `(` right before `max-width`. It should be `and (max-width: 1199px).`

Reply:
Ohhh good 👁️